ingenialink/__init__.py

```diff
@@ -23,12 +23,15 @@
 from .ethercat.network import EthercatNetwork
 from .ethercat.servo import EthercatServo
 
-from .canopen.servo import CanopenServo
-from .canopen.network import CanopenNetwork, CAN_DEVICE, CAN_DEVICE, \
-    CAN_BAUDRATE
-from .canopen.poller import CanopenPoller
-from .canopen.register import CanopenRegister
-from .canopen.dictionary import CanopenDictionary
+try:
+    from .canopen.servo import CanopenServo
+    from .canopen.network import CanopenNetwork, CAN_DEVICE, CAN_DEVICE, \
+        CAN_BAUDRATE
+    from .canopen.poller import CanopenPoller
+    from .canopen.register import CanopenRegister
+    from .canopen.dictionary import CanopenDictionary
```

Comment:
I'm not sure if I like it. What happens if someone tries to call some of these classes, such as CAN_DEVICE, for example?

Reply:
An ImportError will be raised. What is your suggestion to solve this problem?